src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class RightTriangle implements Figure {
+    private String color;
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        this.color = color;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
```

Comment:
There is an unnecessary empty line at the start of the method definition. Please remove it to follow best practices.